tests/helpers.ts

```diff
@@ -1,9 +1,11 @@
+import { APIRequestContext, expect } from '@playwright/test';
 import {
 	DEFAULT_STATUS_LIST_ENCRYPTED_NAME,
 	DEFAULT_STATUS_LIST_INDICES,
 	DEFAULT_STATUS_LIST_PAYMENT_ADDRESS,
 	DEFAULT_SUBJECT_DID,
 	DEFAULT_TESTNET_DID,
+	VERIFICATION_METHOD_TYPES,
```

Comment:
The enumeration exists in the SDK as `VerificationMethods`, as to have a relative declaration with feature synchronisation guarantees.